### PR TITLE
docker_swarm inventory plugin: fix some docs formatting

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Improves docs formatting of `docker_swarm` inventory plugin.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
lib/ansible/plugins/inventory/docker_swarm.py

### CODE REVIEW

File: lib/ansible/plugins/inventory/docker_swarm.py [hunk 1]
```diff
@@ -89,10 +89,10 @@
             default: 60
             aliases: [ time_out ]
         include_host_uri:
-            description: Toggle to return the additional attribute I(ansible_host_uri) which contains the URI of the
-                         swarm leader in format of M(tcp://172.16.0.1:2376). This value may be used without additional
```
Comment (felixfontein) [inline lib/ansible/plugins/inventory/docker_swarm.py:93]: `M(xyz)` creates a link to a module called `xyz`, which totally doesn't make sense here.

**Comment by felixfontein** [comment]
Hmmm, the CI failures (https://app.shippable.com/github/ansible/ansible/runs/118028/62/tests) are totally unrelated to this PR:
```
 failure: Failed to import docker or docker-py (Docker SDK for Python) - No module named docker. Try `pip install docker` or `pip install docker-py` (Python 2.6). 
```
In fact, the real error is a bit earlier: `ERROR! the role 'setup_epel' was not found in /root/.ansible/test/tmp/inventory_docker_swarm-PFYm1A-ÅÑŚÌβŁÈ/test/integration/targets/inventory_docker_swarm/playbooks/roles:/root/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/root/.ansible/test/tmp/inventory_docker_swarm-PFYm1A-ÅÑŚÌβŁÈ/test/integration/targets/inventory_docker_swarm/playbooks`
@mattclay this seems to be related to your recent change (#54917). Do you have any idea what goes wrong here?

**Comment by felixfontein** [comment]
Ok, I've figured it out. The problem is that the tests for the inventory plugin use the `runme.sh` approach, and the playbooks called from there included `setup_docker` via a relative path without changing the role search path. That's now fixed in this PR.

**Comment by felixfontein** [comment]
I've created a new PR for that commit (#54962), I'll rebase this one once that PR is merged.

**Comment by felixfontein** [comment]
ready_for_review

**Comment by WojciechowskiPiotr** [comment]
shipit

**Review by resmo — APPROVED** [review]
shipit

**Comment by felixfontein** [comment]
shipit

**Comment by felixfontein** [comment]
@WojciechowskiPiotr @resmo thanks for reviewing!

**Pull request merged**